.github/workflows/reusable-build-image.yml

```diff
@@ -220,7 +220,7 @@ jobs:
       - generate_matrix
       - build_push
     container:
```

Comment:
We can combine the Cosign step into this job too, right?  
Not for this PR though.